core/tests/protractor_utils/ExplorationEditorHistoryTab.js

```diff
@@ -121,10 +138,10 @@ var ExplorationEditorHistoryTab = function() {
        *                            may be truncated.)
        */
       expectHistoryStatesToMatch: function(expectedStates) {
-        interactionNode.map(function(stateNode) {
+        stateNode.map(function(stateNode) {
```

Comment:
This seems wrong. The item to map over should be a list and named differently from the name given to one of its elements.

Reply:
Looks like I should be more careful with my variable re-naming :smile: . Fixed!